packages/manager/modules/pci/src/projects/project/storages/cold-archive/users/users.less

```diff
@@ -0,0 +1,3 @@
+code.code-break {
+  word-break: break-word;
+}
```

Comment:
**todo:** Use the utility class `word-break` from Bootstrap.

Reply:
I guess, you meant text-break.